src/HotChocolate/Core/src/Types/Types/Scalars/Scalars.cs

```diff
@@ -44,6 +44,8 @@ public static class Scalars
                     typeof(ByteArrayType), TypeContext.None) },
                 { typeof(NameString), new ClrTypeReference(
                     typeof(NameType), TypeContext.None) },
+                { typeof(TimeSpan), new ClrTypeReference(
```

Comment:
@glucaci I have added the type here so that it is recognized by the type system without registering it.